docs/DeveloperGuide.md

```diff
@@ -428,6 +429,43 @@ The `ArgumentMultimap` class is used to parse the user input and store the filte
 
 The `FilterDescriptor` takes in the filter criteria and returns a single predicate encompassing all the criteria on its `getFilters` method, so that this predicate can be used as an argument for the `updateFilteredPersonsList` method of the `Model` object, displaying a list of students that were filtered by this predicate.
 
+### Email Feature
+
+The email mechanism implements the following sequence for the method call execute("email").
+
+#### What is the email feature
+
+The email feature opens up a separate window containing the emails of the current list displayed in Tracey. The email feature allows a user to copy the email list to their clipboard to paste in their email platform of choice.
+
+The `email` command is as follows:
+
+`email`
+
+The user can choose when to execute the email command.
+
+The activity diagram shows the possible execution paths for the `email` command.
+
+**Path Execution of Email Feature Activity Diagram is shown below:**
+![EmailActivityDiagram](images/EmailActivityDiagram.png)
+
+There are two possible execution paths for this command.
+1. User inputs `email` command. After the Email Window opens, the user can choose copy the emails in the list by clicking on the copy email button. After which, the user can close the Email Window.
+2. User inputs `email` command. After the Email Window opens, the user chooses not to copy the emails in the list. After which, the user can close the Email Window.
+
+The sequence diagram below shows the interactions between objects during the execution of a `email` command.
+![EmailSequenceDiagram](images/EmailSequenceDiagram.png)
+
+
+When a user inputs an email command into the Tracey, the `executeCommand()` method of `MainWindow` will be called and this will call the `execute()` method of `LogicManager`. This will trigger a parsing process by `AddressBookParser`,  which then instantiates an `EmailCommand` object. 
+
+Following this, the `LogicManager` will call the `execute()` method of the `EmailCommand` object. In this method, a `CommandResult` object will be instantiated.
+
+Back in the `MainWindow`'s `executeCommand()` method, it will then call the `handleEmailWindow()` method which will then instantiate an `EmailWindow` object.
+
+Afterwards, the `LogicManager` calls the `show()` method of `EmailWindow` and the `EmailWindow` will be shown to the user.
```

Comment:
Very detailed description for developers! Loving it.

Reply:
Thanks 😄